embedded-tests/src/test/java/org/apache/druid/testing/embedded/docker/DruidContainers.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.testing.embedded.docker;
+
+import org.apache.druid.testing.DruidCommand;
+
+/**
+ * Factory for {@link DruidContainerResource} that can run specific services.
+ *
+ * @see #newOverlord()
+ * @see #newCoordinator()
+ */
+public final class DruidContainers
+{
+  private DruidContainers()
+  {
+    // no instantiation
+  }
+
+  /**
+   * Creates a new {@link DruidContainerResource} to run a Coordinator node.
+   */
```

Comment:
nit: this apidoc doesn't add any value beyond the function signature:
```
public static DruidContainerResource newCoordinator()
```

this is true for a lot of places....I think adding apidoc like that could be harmfull as they will become stale after some time.